sql/glam-fenix-dev/glam_etl/org_mozilla_fenix_glam_nightly__scalar_percentiles_v1/query.sql

```diff
@@ -69,7 +69,10 @@ percentiles AS (
 )
 SELECT
   * REPLACE (
-    mozfun.glam.map_from_array_offsets([5.0, 25.0, 50.0, 75.0, 95.0], aggregates) AS aggregates
+    mozfun.glam.map_from_array_offsets_precise(
+      [0.1, 1.0, 5.0, 25.0, 50.0, 75.0, 95.0, 99.0, 99.9],
```

Comment:
This is changing because this query hasn't been recreated after https://github.com/mozilla/bigquery-etl/pull/2723/files